Sitemap/AbstractChain.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+abstract class AbstractChain
+{
+    /**
+     * The list of guessers with sitemap name and priorities as key.
+     *
+     * @var
+     */
+    private $items;
+
+    /**
+     * @param object $item
+     * @param int    $priority
+     * @param string $sitemap
+     */
+    public function addItem($item, $priority = 0, $sitemap = 'default')
+    {
+        if (!isset($this->items[$sitemap])) {
+            $this->items[$sitemap] = array();
+        }
+
+        if (!isset($this->items[$sitemap][$priority])) {
+            $this->items[$sitemap][$priority] = array();
+        }
+
+        $this->items[$sitemap][$priority][] = $item;
+    }
+
+    /**
+     * @param $sitemap
+     *
+     * @return array
+     */
+    protected function getSortedItemsBySitemap($sitemap)
```

Comment:
i think this should be called "getSortedItemsForSitemap". by sitemap made me think that the items are sorted by sitemap. but we only ever get items for one specific sitemap